src/onboarding/Templates/TemplateDetails.js

```diff
@@ -209,47 +208,44 @@ function TemplateDetails({ template, visible, onUse, onClose }) {
                 height: 150px;
               `}
             >
-              {template.optionalApps.map(({ appName, label }, index) => (
-                <div
-                  key={index}
-                  css={`
-                    display: flex;
-                    justify-content: space-between;
-                    margin-top: ${2 * GU}px;
-
-                    & + & {
-                      margin-top: ${1.5 * GU}px;
-                    }
-                  `}
-                >
-                  <KnownAppBadge appName={appName} label={label} />
+              {() =>
+                template.optionalApps.map(({ appName, label }, index) => (
                   <div
+                    key={index}
                     css={`
                       display: flex;
-                      align-items: center;
-                      ${unselectable}
+                      justify-content: space-between;
+                      margin-top: ${2 * GU}px;
+
+                      & + & {
+                        margin-top: ${1.5 * GU}px;
+                      }
                     `}
                   >
-                    <Checkbox
-                      checked={templateOptionalApps[appName]}
-                      onChange={() => {
-                        setTemplateOptionalApps(apps => ({
-                          ...apps,
-                          [appName]: !apps[appName],
-                        }))
-                      }}
+                    <KnownAppBadge appName={appName} label={label} />
+                    <label
                       css={`
-                        margin-right: ${1.5 * GU}px;
-                        border-color: ${theme.hint};
```

Comment:
Just a note on this, @dizzypaty and I had a _very_ quick discussions on using these custom borders because the default was quite difficult to make out against the background:

<img width="94" alt="Screen Shot 2019-10-10 at 11 07 16 AM" src="https://user-images.githubusercontent.com/4166642/66555003-3de67280-eb4e-11e9-8a15-4fc6303cf386.png">

Reply:
Ah yes, it’s clearly not contrasted enough. We should find a color that works for both the background and the base surface for these (and Radio as well). /cc @owisixseven

In the meantime, we could add the color again (and if possible, another one for the pressed state), but I would avoid using `hint` here: this color is not meant to be used for borders, and it might not work in the dark theme or in a future version of the light theme. I think we could use the hex code directly to make these kind of changes: it makes it easy to identify that we should update it in the future.